server/tests/integration/server.int.test.js

```diff
@@ -150,7 +150,9 @@ describe(atEndpoint, () => {
 
 describe(`${testEndpoint}/import`, () => {
     it(`POST ${testEndpoint}/import`, async () => {
-        const response = await request(listener).post(`${testEndpoint}/import`).send({git_hash: 1234});
+        const response = await request(listener)
+            .post(`${testEndpoint}/import`)
+            .send({ git_hash: 1234 });
         expect(response.statusCode).toBe(200);
```

Comment:
These changes were made in https://github.com/bocoup/aria-at-app/pull/146/files#diff-15c43bcc258fd5c8ff9f71814f5658a1

Reply:
@s3ththompson could you take care of this part?

Reply:
done!